@Types/ITechRecord.d.ts

```diff
@@ -124,6 +124,8 @@ export default interface ITechRecord {
   manufacturerDetails: ManufacturerDetails;
   authIntoService: AuthIntoService;
   lettersOfAuth: LettersOfAuth;
+  historicPrimaryVrm?: string;
+  historicSecondaryVrms?: string;
```

Comment:
as this is plural, could this be an array type?

Reply:
Yes it should be you're absolutely right